docker/store_query_producer.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+## Install the `dig` command
+apk --no-cache add bind-tools
```

Comment:
The file name is strange - what is this script "producing"? Is it meant as "producing the store query message"?

Second question - how long does it take to start this one and the publisher? It feels weird to have to install the deps everytime you want to (re)start the test. Maybe add a dockerfile with the packages installation and use an image built from that?

Reply:
Thanks for the comments!

The `store_query_producer.sh` just produces _Store_ requests. So yes, is aimed to generate "store query messages" so that we have traffic from the message retrieval.

https://github.com/waku-org/nwaku/blob/4539dfc76109382f321080fdfbf92e44531c6de8/waku/node/waku_node.nim#L610

That makes me think that I'm missing a configuration setting for the nodes, which is the `storenode` param.

The main purpose of this script is to force the `nwaku` node, to perform a query request to the other node.

I'm open to renaming it to a more appropriate name.

Re the second question, these scripts are fast to start, even having those installations. The thing is that I hard-coded a 20s wait to make sure the `nwaku` nodes had started completely.